docs/pages/admin-guides/access-controls/access-request-plugins.mdx

```diff
@@ -12,15 +12,15 @@ With Teleport's Access Request plugins, users can manage Access Requests from
 within your organization's existing messaging and project management solutions.
 
 Access Request plugins are self-contained programs that connect to the Teleport
-Auth Service's gRPC API to listen for audit events relating to new or updated
+Auth Service instances gRPC API to listen for audit events relating to new or updated
```

Comment:
Wouldn't it be correct to say "Teleport Auth Service gRPC API"? Teleport Auth Service describes a type of service available in Teleport. It doesn't seem like adding "instances" adds much value here. Well, unless it's solely to satisfy the linter.

Reply:
Yeah, I think this should be "Auth Service gRPC API", since the API belongs to the service itself, rather than any particular instance. The linter looks for `Auth Services` ([GitHub](https://github.com/gravitational/teleport/blob/7086f7a126480f31b7519b0aa293204e30e53ed4/.github/vale-styles/messaging/consistent-terms.yml#L19)), so it shouldn't have a problem with the apostrophe.

Reply:
Corrected these entries